client/my-sites/backup/backup-contents-page/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import DocumentHead from 'calypso/components/data/document-head';
+import ActionButtons from 'calypso/components/jetpack/daily-backup-status/action-buttons';
+import cloudIcon from 'calypso/components/jetpack/daily-backup-status/status-card/icons/cloud-success.svg';
+import useGetDisplayDate from 'calypso/components/jetpack/daily-backup-status/use-get-display-date';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import Main from 'calypso/components/main';
+import SidebarNavigation from 'calypso/components/sidebar-navigation';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+import { useSelector } from 'calypso/state';
+import isJetpackSiteMultiSite from 'calypso/state/sites/selectors/is-jetpack-site-multi-site';
+import './style.scss';
+
+interface OwnProps {
+	rewindId: number;
+	siteId: number;
+}
+
+const BackupContentsPage: FunctionComponent< OwnProps > = ( { rewindId, siteId } ) => {
+	const translate = useTranslate();
+	const getDisplayDate = useGetDisplayDate();
+	const moment = useLocalizedMoment();
+	const displayDate = getDisplayDate( moment.unix( rewindId ), false );
+
+	const isMultiSite = useSelector( ( state ) => isJetpackSiteMultiSite( state, siteId ) );
+
+	return (
+		<>
+			<Main className="main backup-contents-page">
+				<DocumentHead title={ translate( 'Backup contents' ) } />
+				{ isJetpackCloud() && <SidebarNavigation /> }
+				<Card className="daily-backup-status contents-page">
+					<div className="contents-page__header">
```

Comment:
Should the class name prefix be `backup-contents-page` everywhere instead of just `contents-page`?

Reply:
@jwebbdev Thanks! Made a bit of mess with that trying to reuse the CSS from daily-backup-status. Just fixed the classNames and removed one that wasn't being in use. Also corrected the SCSS file. Could you re-review it?